src/components/Record.js

```diff
@@ -3,13 +3,22 @@ import moment from 'moment';
 export default function Record({ className, title, createdAt, type, text }) {
     const formattedTime = moment(createdAt).calendar();
 
+    let renderedText = text;
+    if (text.startsWith('http')) {
+        renderedText = (
+            <a className="link" href="text">
```

Comment:
Does this need to read `<a className="link" href={text}`?

Reply:
😮 nice catch! yes, it does...

Reply:
updated - mind re-reviewing?